### PR TITLE
add contributors to FAQ page

### PR DESCRIPTION
**Pull request by saritadc**
added contributors to the FAQ page.

### CODE REVIEW

**Comment by shiltemann** [comment]
Thanks for your contribution @saritadc! We don't track contributors of FAQs, only tutorials and slides, so this will not display any contributors currently.

If you would like to work on FAQs, we could use somebody who walks through all of them to check if they are still up-to-date with the latest version of Galaxy, since the FAQs cover frequently used tasks in the interface, which changes a lot. 

A slightly bigger project here would be to support multiple versions of the FAQs, connected to differenct Galaxy versions, and the ability to switch between them. But we would need to think together about how to do that a bit first.

**Comment by hexylena** [comment]
@shiltemann this is to fix https://github.com/galaxyproject/training-material/issues/2973

File: _layouts/faq.html [hunk 1]
```diff
@@ -2,10 +2,18 @@
...
+    <div class="contributors-line">{{ locale['authors'] | default: "Authors" }}: {% include
+        _includes/contributor-list.html
+        contributors=page.contributors badge=true newcontributors=true %}</div>
+  {% endif %}
+
+    {{ page.description }}
+    <hr>
 
-{{content | regex_replace: '<hr.*—<\/p>', '' }}
+    {{content | regex_replace: '<hr.*—<\ /p>', '' }}
```
Comment (hexylena) [inline _layouts/faq.html:17]:
```suggestion
    {{content | regex_replace: '<hr.*—<\/p>', '' }}
```

I don't think this can have a space here.

**Review by hexylena — COMMENTED** [review]
This looks good @saritadc I've tested locally. Would you like to fix the one review comment and also maybe add yourself to the CONTRIBUTORS.yaml file as well? And then I think I'm happy to merge.

**Comment by saritadc** [comment]
@hexylena I have fixed the review comment and I think it's good to go. Thank you!

**Review by hexylena — APPROVED** [review]

**Pull request merged**